core/src/env/api.rs

```diff
@@ -238,25 +238,6 @@ pub fn clear_contract_storage(key: &Key) {
     })
 }
 
-/// Invokes a call to the runtime.
-///
-/// # Note
-///
-/// The call is not guaranteed to execute immediately but might be deferred
-/// to the end of the contract execution.
-///
-/// # Errors
-///
-/// - If the called runtime function does not exist.
-pub fn invoke_runtime<T>(params: &T::Call) -> Result<()>
-where
-    T: EnvTypes,
-{
-    <EnvInstance as OnInstance>::on_instance(|instance| {
-        TypedEnv::invoke_runtime::<T>(instance, params)
-    })
-}
-
```

Comment:
So we decided on the "delete" vs feature-flag (which is what I prefer too). Could do this as a discrete PR then bringing them back could be as simple as a `git revert`

Reply:
We won't bring them back since we now instead have
```rust
pub fn ext_call_chain_extension(
    func_id: u32,
    input_ptr: u32,
    input_len: u32,
    output_ptr: u32,
    output_len_ptr: u32,
) -> u32;
```

Reply:
Perfect :+1: